terraform/eks/main.tf

```diff
@@ -227,7 +227,7 @@ module "validator" {
   cortex_instance_endpoint = var.cortex_instance_endpoint
   rollup                   = var.rollup
   kubecfg_file_path        = abspath("./${local_file.kubeconfig.filename}")
-  k8s_deployment_name      = var.aoc_base_scenario == "otlp" ? module.aoc_oltp.0.sample_app_deployment_name : ""
+  k8s_deployment_name      = var.aoc_base_scenario == "oltp" ? module.aoc_oltp[0].sample_app_deployment_name : ""
```

Comment:
Can this be changed to aoc_otlp

Reply:
I will change that when I bring in changes from `terraform` branch in a separate PR.